worker-document/src/main/java/com/hpe/caf/worker/document/tasks/AbstractTask.java

```diff
@@ -67,6 +67,21 @@ public final WorkerResponse createWorkerResponse()
     @Nonnull
     protected abstract WorkerResponse createWorkerResponseImpl();
 
+    /**
+     * Returns appropriate WorkerResponse for the task in the event of a general task failure.
+     *
+     * @param failure Detail of the failure that occurred.
+     * @return A WorkerResponse with task appropriate failure details.
```

Comment:
A very minor point which we rarely follow but I'll point it out just FYI and since it is largely followed in this project, particularly in worker-document-interface which is the external JavaDoc:

The param and return blocks should not really be full sentences starting with a capital letter and ending in a full-stop.  They are meant to be phrases.